### PR TITLE
Document `share_bones_time_early`

### LINKED ISSUE #819: share_bones_time_early is not documented in the game API

### PR DESCRIPTION
**Pull request by red-001**
Fixes #819.

### CODE REVIEW

**Comment by paramat** [comment]
Please split the line into 2 roughly equal length lines.
Also for better English:
"How much earlier the bones of a dead player can be looted by
everyone if the player dies in a protected area they don't own."

**Comment by red-001** [comment]
@paramat done.

File: minetest.conf.example [hunk 1]
```diff
@@ -9,6 +9,11 @@
 # 0 to disable
 #share_bones_time = 1200
 
+# How much earlier can the bones of a dead playerbe looted
+# by everyone if the player dies in a protected area they don't own.
+# 0 to disable. 
+# share_bones_time_early = share_bone_time / 4
```
Comment (kaeza) [inline minetest.conf.example]: This should be set to 300, and the "default" of 1/4 moved to description. Also, remove the space between the `#` and the setting name.

**Comment by red-001** [comment]
@kaeza done.

**Comment by paramat** [comment]
```How much earlier can the bones of a dead player be looted```
Better English needed, move the 'can' as in my previous comment.

**Comment by paramat** [comment]
^ @red-001

**Comment by paramat** [comment]
Thanks please squash then i'll merge.

**Comment by red-001** [comment]
@paramat Done.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
3a98245f62aeabdd1bc4ac5a83441cf6aec3cf88

**Pull request closed without merging**